TransformManager.cs

```diff
@@ -142,203 +129,188 @@ public void SetActiveGizmoType(string sType)
-        {
-            get { return activeGizmo; }
-        }
+        public ITransformGizmo ActiveGizmo => _activeGizmo;
 
         public event EventHandler OnActiveGizmoModified;
-        protected virtual void SendOnActiveGizmoModified() {
-            var tmp = OnActiveGizmoModified;
-            if (tmp != null)
-                tmp(this, new EventArgs());
-        }
-
 
+        protected virtual void SendOnActiveGizmoModified()
+        {
+            EventHandler tmp = OnActiveGizmoModified;
+            tmp?.Invoke(this, new EventArgs());
+        }
 
         public FrameType ActiveFrameType
         {
-            get {
-                return (activeGizmo != null && activeGizmo.SupportsFrameMode) ? 
-                    activeGizmo.CurrentFrameMode : defaultFrameType;
+            get
+            {
+                return _activeGizmo != null && _activeGizmo.SupportsFrameMode
+                    ? _activeGizmo.CurrentFrameMode
+                    : _defaultFrameType;
             }
-            set {
-                if (activeGizmo != null && activeGizmo.SupportsFrameMode) {
-                    activeGizmo.CurrentFrameMode = value;
-                    if ( activeGizmo.Targets.Count == 1 )
-                    lastFrameTypeCache[activeGizmo.Targets[0]] = value;
+            set
+            {
+                if (_activeGizmo != null && _activeGizmo.SupportsFrameMode)
+                {
+                    _activeGizmo.CurrentFrameMode = value;
+                    if (_activeGizmo.Targets.Count == 1)
+                    {
+                        _lastFrameTypeCache[_activeGizmo.Targets[0]] = value;
+                    }
                     //defaultFrameType = value;       // always change default when we explicitly change type
                     SendOnActiveGizmoModified();
-                } else {
-                    defaultFrameType = value;
-                    SendOnActiveGizmoModified();        // not really right...but UI using it now
+                }
+                else
+                {
+                    _defaultFrameType = value;
+                    SendOnActiveGizmoModified(); // not really right...but UI using it now
                 }
             }
         }
 
-
         public void SetActiveReferenceObject(SceneObject so)
         {
-            if (activeGizmo != null && activeGizmo.SupportsReferenceObject)
-                activeGizmo.SetReferenceObject(so);
+            if (_activeGizmo != null && _activeGizmo.SupportsReferenceObject)
+            {
+                _activeGizmo.SetReferenceObject(so);
+            }
         }
 
-
         public void DismissActiveGizmo()
         {
             FScene scene = Context.Scene;
-            if ( activeGizmo != null ) {
-                scene.RemoveUIElement(activeGizmo, true);
-                activeGizmo = null;
+            if (_activeGizmo != null)
+            {
+                scene.RemoveUIElement(_activeGizmo, true);
```

Comment:
need to add named argument to second parameter "true"

Reply:
done